### PR TITLE
Merged suggestions

### PR DESCRIPTION
**Pull request by dgarcia360**

### CODE REVIEW

File: whitepaper/chapters/blockchain.tex [hunk 1]
```diff
@@ -226,7 +226,7 @@ \subsection{Automatic Delegated Harvester Detection}
 
 When \nemsetting{user}{enableDelegatedHarvestersAutoDetection} is set, the server allows other accounts to register as delegated harvesters via special transfer messages.
 The server inspects all transfer messages sent to its \nemsetting{user}{bootPrivateKey} account.
-Each message that begins with the magic bytes \texttt{0x98E5BF64C771CCFE} is written out to a file queue for further processing.
+Each message that begins with the magic bytes \texttt{0xE201735761802AFE} is written out to a file queue for further processing.
```
Comment (dgarcia360) [inline whitepaper/chapters/blockchain.tex:239]: @Jaguar0625 Please double check if this is correct before merging

File: whitepaper/chapters/timesync.tex [hunk 2]
```diff
@@ -97,7 +97,7 @@ \subsection{Calculation of the Effective Offset}
 This gives the formula for the effective offset $o$
 $$ o = \mathvar{scale} \sum_j I_j \: o_j$$
 
-Note that the influence of an account with large importance is artificially limited because the term $\frac{n}{s} $ caps the scale.
+Note that the influence of an account with large importance is artificially limited because the term $\frac{s}{n} $ caps the scale.
```
Comment (dgarcia360) [inline whitepaper/chapters/timesync.tex]: @Jaguar0625 Please double check if this is correct before merging

File: whitepaper/chapters/accounts.tex [hunk 3]
```diff
@@ -19,15 +19,15 @@ \subsection{Addresses}
 \end{itemize}
 
 The checksum allows for quick recognition of mistyped addresses.
-It is possible to send mosaics to any valid address even if the address has not previously participated in any transaction.
+It is possible to send mosaics\footnote{A mosaic is a digital asset defined on Symbol. Other technologies refer to mosaics as tokens.} to any valid address even if the address has not previously participated in any transaction.
```
Comment (Jaguar0625) [inline whitepaper/chapters/accounts.tex]: Symbol => \codename

File: whitepaper/chapters/blockchain.tex [hunk 4]
```diff
@@ -113,12 +113,12 @@ \subsection{Block Generation}
 As a consequence, this strategy is rarely profit maximizing for the harvester.}
 \item{\texttt{maximize-fee}: \\
 Transactions are selected in such a way that the cumulative fee for all transactions in a block is maximized.
-A greedy node will pick this strategy.
+A profit-centric node will pick this strategy.
```
Comment (Jaguar0625) [inline whitepaper/chapters/blockchain.tex]: centric => optimizing ?

File: whitepaper/chapters/blockchain.tex [hunk 5]
```diff
@@ -113,12 +113,12 @@ \subsection{Block Generation}
...
 Maximizing the total block fee does not necessarily mean that the number of transactions included is maximized as well.
 In fact, in many cases, the harvester will only include a subset of the transactions that are available.}
 \item{\texttt{minimize-fee}: \\
 Transactions with the lowest maximum fee multipliers are selected first by this strategy.
-Generous nodes will pick this strategy together with a very low \nemsetting{node}{minFeeMultiplier}.
+Service-centric nodes will pick this strategy together with a very low \nemsetting{node}{minFeeMultiplier}.
```
Comment (Jaguar0625) [inline whitepaper/chapters/blockchain.tex]: Altruistic ?

File: whitepaper/chapters/blockchain.tex [hunk 6]
```diff
@@ -127,8 +127,8 @@ \subsection{Block Generation}
 
 Transactions can initiate transfers of both static and dynamic amounts.
 Static amounts are fixed and independent of external factors.
-For example, the amount specified in a transfer transaction is static.
-The exact amount specified is always transfered from sender to recipient.
+For example, the amount specified in a\nemtechdocsfootnote{concepts/transfer-transaction.html} is static.
```
Comment (Jaguar0625) [inline whitepaper/chapters/blockchain.tex]: missing space after a

File: whitepaper/chapters/blockchain.tex (hunk 1 above)
Comment (Jaguar0625) [reply whitepaper/chapters/blockchain.tex:239]: yes it is correct

File: whitepaper/chapters/consensus.tex [hunk 7]
```diff
@@ -347,7 +347,7 @@ \subsubsection*{Small Account}
 The high fees paid boost the account’s importance enough so that it is able to harvest at least one block per importance recalculation interval.
 From this point forward, the account behaves like the large account in the previous section.
 It will also add a transaction with a high fee to one of its own harvested blocks each recalculation interval.
-The account hopes that due to the increased probability of harvesting a block, its additional collected fees will exceed its costs.
+The account operator hopes that due to the increased probability of harvesting a block, its additional collected fees will exceed its costs.
```
Comment (Jaguar0625) [inline whitepaper/chapters/consensus.tex]: operator => owner (not sure)?

File: whitepaper/chapters/disruptor.tex [hunk 8]
```diff
@@ -11,7 +11,7 @@ \section{Disruptor}
 New elements are inserted into the next free slot of the ring buffer.
 Fully processed elements are removed to make space for new elements.
 Since the ring buffer has a finite number of slots, it can run out of space if processing can't keep up with new insertions.
-The behavior of \codename, in such a case, can be configured to either to exit the server or discard new data until a slot becomes available.
+The behavior of \codename, in such a case, can be configured either to exit the server or discard new data until a slot becomes available.
```
Comment (Jaguar0625) [inline whitepaper/chapters/disruptor.tex]: i think you removed the wrong to
> can be configured to either exit


reads more natural to me

File: whitepaper/chapters/timesync.tex [hunk 9]
```diff
@@ -35,7 +35,7 @@ \subsection{Gathering Samples}
 
 After the start up of a node is completed, the node (hereafter called \emph{local node}) selects partner nodes for performing a time synchronization round.
 
-For all selected partners, the local node sends out a request asking the partner for its current network time.
+For all selected partners, the local node sends out a request asking the partners for their current network time.
```
Comment (Jaguar0625) [inline whitepaper/chapters/timesync.tex]: consider:

For each selected partners, the local node sends out a request asking the partner for its current network time

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]

File: whitepaper/chapters/timesync.tex (hunk 2 above)
Comment (Jaguar0625) [reply whitepaper/chapters/timesync.tex]: this is wrong. in the equation `1/(s/n))` == `n/s`

File: whitepaper/chapters/timesync.tex (hunk 9 above)
Comment (Jaguar0625) [reply whitepaper/chapters/timesync.tex]: selected partner (no s) - sorry i had a typo before

**Review by gimre-xymcity — DISMISSED** [review]
besides the fraction issue mentioned - looks good

**Pull request merged**